skyplane/compute/gcp/gcp_auth.py

```diff
@@ -59,7 +59,7 @@ def save_region_config(discovery, self):
                             continue
                         region_name = region["name"]
                         for quota_item in region["quotas"]:
-                            if quota_item["metric"] == "CPUS":
+                            if quota_item["metric"] == "N2_CPUS":  # Initially only concerned with N2 limits
```

Comment:
Since we are by default using N2 instances as a v1 for this PR, changing the quota info to "N2_CPUS" is more accurate. However, in the future, we can save quotas for all family types and make decisions based on that.